apps/next-example/babel.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
         alias: {
           'react-native': '../../node_modules/react-native-web',
           // Uncomment this if you want fast-refresh to work with reanimated:
-          // 'react-native-reanimated': '../src/index',
```

Comment:
What's this?

Reply:
This is an incorrect path 😄

Reply:
I think it wasn't updated after migrating to monorepo. There is no src directory in any of ancestor directories relative to the `next-example`.